draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -805,12 +805,18 @@ a different salt.
 
 ## Choice of a Hash Algorithm
 
-For the security of this scheme, the hash algorithm is required to be preimage and collision
+For the security of this scheme, the hash algorithm is required to be first and second preimage
```

Comment:
Is first and second preimage resistance a common term? I usually see "preimage resistant" and "second-preimage resistant".

Reply:
```suggestion
For the security of this scheme, the hash algorithm is required to be preimage resistant and second-preimage 
```